frontend/IndexPicante.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://unpkg.com/tailwindcss@2.2.4/dist/tailwind.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+</head>
+<body>
+    <!-- me tiene que llegar del back -->
+    <script>
+    const listas = [
+        {
+            id: "1",
+            titulo: "Lista 1",
+        },
+        {
+            id: "2",
+            titulo: "Lista 2",
+        },
+        {
+            id: "3",
+            titulo: "Lista 3",
+        },
+        {
+            id: 4,
+            titulo: "Lista 1",
+        },
+        {
+            id: 5,
+            titulo: "Lista 1",
+        },
+        {
+            id: 6,
+            titulo: "Lista 1",
+        },
+        {
+            id: 7,
+            titulo: "Lista 1",
+        },
+        {
+            id: 8,
+            titulo: "Lista 1",
+        },
+        {
+            id: 9,
+            titulo: "Lista 1",
+        },
+        {
+            id: 10,
+            titulo: "Lista 1",
+        },
+        {
+            id: 11,
+            titulo: "Lista 1",
+        },
+        {
+            id: 12,
+            titulo: "Lista 1",
+        },
+        {
+            id: 13,
+            titulo: "Lista 1",
+        },
+    ];
+
+    //listas = [];
+
+    </script>
+
+    <!-- titulo de pagina -->
+    <div class="flex bg-green-100 ml-4 mr-4">
+        <div class="w-1/6 text-center place-self-center">
+            <button class="ml-3 p-1 border-2 border-green-500 text-green-800 bg-green-100 hover:bg-green-200 rounded">
+                <i class="bi bi-plus-lg" style="font-size: 1.5rem;"></i>
+            </button>
+        </div>
+        <div class="w-5/6 text-center place-self-center mr-5 sm:mr-16 md:mr-24 lg:mr-44">
+            <p class="text-6xl text-center bg-green-100 ml-4 mr-4 mt-4 mb-4">
+                To do list
+            </p>
+        </div>
+    </div>
+
+    <hr class="border-1 border-black ml-4 mr-4 mt-2 mb-4">
+
+    <div id="grilla">
+        
+        <script>
```

Comment:
Use js in one place. move this to multiple files

Reply:
i already moved it to an js file